src/atoms/TextInput/TextInput.web.js

```diff
@@ -284,10 +311,12 @@ const TextInput = ({
     }
   }, [inputRef, containerRef]);
 
+  const _rows = variant === 'outlined' ? 1 : 3;
```

Comment:
What is this constraint? Is it in design?

Reply:
had to add _rows=1 for textarea with outlined variant to have its initial size smaller. for variant `filled` 3 rows what comes under max-height of 64px.